azure-commands/connector.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+
+from .operations import operations, _check_if_right_user, _login_az_cli
+
+logger = get_logger('azure-commands')
+
+
+class AzureCommandsConnector(Connector):
```

Comment:
Class name can be: AzureCommandsConnector => AzureCommands

Reply:
Changed.